src/share/streams/shr_strdata_mod.F90

```diff
@@ -191,7 +190,6 @@ subroutine shr_strdata_init(SDAT,mpicom,compid,name,scmmode,scmlon,scmlat, &
   integer(IN) :: nu,nv         ! u,v index
   integer(IN) :: my_task,npes  ! my task, total pes
   integer(IN),parameter :: master_task = 0
-  character(CS) :: lname       ! local name
```

Comment:
Since it looks like the name optional argument is unused, it should be removed as an argument to this routine to avoid confusion